_data/administrations/provincial.yml

```diff
@@ -3,62 +3,50 @@
   name:
     en: Province of British Columbia
     fr: Province de la Colombie-Britannique
-
```

Comment:
Removed spaces because it was causing unnecessary diffs on commits when creating a new organization, same goes for the comment. It shouldn't be an issue for federal administration since we can't currently add a federal administration, but still changed it for consistency.